src/components/Header/TopNav/ServicesSpecialitiesMap.js

```diff
@@ -7,34 +9,43 @@ const specialitiesMap = {
   openSource: []
 }
 
-const servicesColors = {
-  default: 'black',
-  defaultText: 'white',
+const logoColors = {
+  default: theme.colors.text,
+  defaultText: theme.colors.white,
   defaultHover: '#8e8e8e',
-  engineering: '#52FFAC',
-  design: '#52FFAC',
-  training: '#52FFAC',
-  delivery: '#52FFAC',
-  dedicatedTeams: '#52FFAC',
-  openSource: '#52FFAC',
-  specialityText: '#090329',
-  specialityHover: 'white'
+  engineering: {
+    'node-js': '#52FFAC',
+    graphql: '#EB008B',
+    'vue-js': '#039328',
+    'react-js': '#0BDDF9',
+    kubernetes: '#29EFEF'
+  },
+  design: {},
+  training: {},
+  delivery: {},
+  dedicatedTeams: {},
+  openSource: {},
```

Comment:
do we need any of these?

Reply:
Not yet - but this was part of a previous ticket to have these services defined somewhere.
So far only the first 3 are live on our website (and only engineering has specialities pages. They all will have in the future)